data/sql/updates/pending_db_world/rev_1643309943188748904.sql

```diff
@@ -0,0 +1,74 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1643309943188748904');
+
+-- update current loot to 100% per wowhead
+UPDATE `gameobject_loot_template` SET `Chance`='100' WHERE  `Entry`=11524 AND `Item`=11614;
+UPDATE `gameobject_loot_template` SET `Chance`='100' WHERE  `Entry`=11525 AND `Item`=11615;
+UPDATE `gameobject_loot_template` SET `Chance`='100' WHERE  `Entry`=13721 AND `Item`=12827;
+UPDATE `gameobject_loot_template` SET `Chance`='100' WHERE  `Entry`=13722 AND `Item`=12830;
+
+-- delete and insert for rerun locking the loot to skill id requirement for loot
+-- none currently exist in acdb
+DELETE FROM `conditions` WHERE `SourceTypeOrReferenceId`=4 AND `SourceGroup`IN ( 11524, 11525, 13721, 13722);
+INSERT INTO `conditions` SET 
```

Comment:
these inserts makes no sense.

Reply:
right now i am waiting for more feed back on if the darn conditions are even working as intended right now.